low/low.go

```diff
@@ -780,3 +784,11 @@ func SetCountersEnabledInApplication(enabled bool) {
 func SetNextMbuf(next *Mbuf, prev *Mbuf) {
 	prev.next = (*C.struct_rte_mbuf)(next)
 }
+
+func GetPacketOffloadFlags(mb *Mbuf) uint64 {
+	return uint64(mb.ol_flags)
+}
+
+func GetPacketTimestamp(mb *Mbuf) uint64 {
```

Comment:
Could you please add this function to our wiki "Helper API" -> "All packets" -> "Timestamps" or so